package.json

```diff
@@ -277,7 +277,7 @@
     "@types/react-redux": "6.0.4",
     "@types/redux-logger": "^3.0.7",
     "awesome-typescript-loader": "^5.2.1",
-    "brave-ui": "github:brave/brave-ui#e0aff04ebfa2b4ce03043f31a06d6127afb17bea",
+    "brave-ui": "github:emerick/brave-ui#twitter-tips",
```

Comment:
don't forget to update this to the latest brave-ui master when UI PR is merged

Reply:
Thanks for the reminder!